tests/console-backend-service/internal/domain/k8s/namespace_test.go

```diff
@@ -25,32 +23,46 @@ func TestNamespace(t *testing.T) {
 
 	suite := givenNewTestNamespaceSuite(t)
 
+	t.Log("Subscribing to namespaces...")
+	subscription := suite.whenNamespacesAreSubscribedTo()
+	defer subscription.Close()
+
 	t.Log("Creating namespace...")
 	createRsp, err := suite.whenNamespaceIsCreated()
 	suite.thenThereIsNoError(t, err)
 	suite.thenThereIsNoGqlError(t, createRsp.GqlErrors)
 	suite.thenCreateNamespaceResponseIsAsExpected(t, createRsp)
 	suite.thenNamespaceExistsInK8s(t)
+	suite.thenAddEventIsSent(t, subscription)
 
 	t.Log("Quering for namespace...")
 	queryRsp, err := suite.whenNamespaceIsQueried()
 	suite.thenThereIsNoError(t, err)
 	suite.thenThereIsNoGqlError(t, queryRsp.GqlErrors)
 	suite.thenNamespaceResponseIsAsExpected(t, queryRsp)
 
+	t.Log("Quering for namespaces...")
+	listQueryRsp, err := suite.whenNamespacesAreQueried()
+	suite.thenThereIsNoError(t, err)
+	suite.thenThereIsNoGqlError(t, listQueryRsp.GqlErrors)
+	suite.thenNamespacesResponseIsAsExpected(t, listQueryRsp, queryRsp)
+
 	t.Log("Updating namespace...")
 	updateResp, err := suite.whenNamespaceIsUpdated()
 	suite.thenThereIsNoError(t, err)
 	suite.thenThereIsNoGqlError(t, updateResp.GqlErrors)
 	suite.thenUpdateNamespaceResponseIsAsExpected(t, updateResp)
 	suite.thenNamespaceAfterUpdateExistsInK8s(t)
+	suite.thenUpdateEventIsSent(t, subscription)
 
 	t.Log("Deleting namespace...")
 	deleteRsp, err := suite.whenNamespaceIsDeleted()
 	suite.thenThereIsNoError(t, err)
 	suite.thenThereIsNoGqlError(t, deleteRsp.GqlErrors)
 	suite.thenDeleteNamespaceResponseIsAsExpected(t, deleteRsp)
 	suite.thenNamespaceIsRemovedFromK8sEventually(t)
+	//namespace changes its status to 'Terminating' first - delete event is sent after few seconds
+	suite.thenUpdateEventIsSent(t, subscription)
```

Comment:
Can we check if it works as expected for pods? Most simplistic scenario, create a pod which will stop just after startup. If it's too complicated leave it out. We have it covered by unit tests for now.